tests/subworkflows/nf-core/vcf_impute_glimpse/main.nf

```diff
@@ -36,3 +36,34 @@ workflow test_vcf_impute_glimpse_without_sample {
 workflow test_vcf_impute_glimpse_with_sample {
     VCF_IMPUTE_GLIMPSE ( input_vcf_with_samples_infos, ch_ref_panel, ch_map )
 }
+
+workflow test_vcf_impute_glimpse_multiple {
+    region        = Channel.fromList(["chr21:16600000-16790000", "chr21:16610000-16800000"])
+    input_vcf     = Channel.fromList([
+        [[ id:'input'], // meta map
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/glimpse/NA12878.chr21.s.1x.vcf.gz", checkIfExists: true),
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/glimpse/NA12878.chr21.s.1x.vcf.gz.csi", checkIfExists: true)],
+        [[ id:'input2'], // meta map
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/glimpse/NA12878.chr21.s.1x.vcf.gz", checkIfExists: true),
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/glimpse/NA12878.chr21.s.1x.vcf.gz.csi", checkIfExists: true)]
+    ])
+    input_vcf_multiple    = input_vcf
+        .combine(Channel.of([[]]).collect())
+        .combine(region)
+        .map{ metaI, vcf, index, sample, region ->
+            [metaI + ["region": region], vcf, index, sample, region]}
+        .view()
+    ch_ref_panel = Channel.fromList([
+        [[ ref:'ref_panel'],
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/glimpse/1000GP.chr21.noNA12878.s.bcf",
+            checkIfExists: true),
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/glimpse/1000GP.chr21.noNA12878.s.bcf.csi",
+            checkIfExists: true)],
+        [[ ref:'ref_panel2'],
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/glimpse/1000GP.chr21.noNA12878.s.bcf",
+            checkIfExists: true),
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/glimpse/1000GP.chr21.noNA12878.s.bcf.csi",
+            checkIfExists: true)]
+    ])
+    VCF_IMPUTE_GLIMPSE ( input_vcf_multiple, ch_ref_panel, ch_map )
```

Comment:
Could you add a test where the `input_vcf_multiple` channel contains VCFs from at least two different chromosomes and the `ch_ref_panel ` and `ch_map` has also the reference/map from these two different chromosomes?

Reply:
I will need to update the dataset accordingly to do so...
To keep the dataset small I will focus on 200 000 bases, would it be enough for you ?

Reply:
It seems that glimpse1_phase cannot take a map on multiple chromosome.
I will need to parallelize on the chr for everything.
It will also be easier to set up afterwards.